src/test/resources/elasticsearchdata/anne-gramsci-n02.json

```diff
@@ -47,7 +47,7 @@
     "immigrationStatus": "string",
     "nationality": "string",
     "notes": "string",
-    "offenderDetails": {},
+    "offenderDetails": "string",
```

Comment:
This field was switched from a Map to a String two years ago; test data finally catching up